doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+
+Flexible and Low-Latency network topologies
+   DECT NR+ supports various network configurations, like star networks or mesh networks, adapting to specific applications.
+
+High Reliability
+   DECT NR+ employs Automatic Repeat Request (ARQ) to correct data transmission errors, ensuring reliable communication for mMTC applications.
+
+Security
+   Possibility of hiding the network using AES-128 encryption and integrity protection.
+   DECT NR+ communication can be encrypted and the network itself hidden for enhanced security.
+
+High-speed
+   DECT NR+ reaches data rates up to 3.4 Mbps, depending on modulation.
+   DECT NR+ allows selecting the modulation scheme for high-speed communication suitable for various applications.
+
+Comparison of DECT NR+ and LTE
+******************************
+
+While both DECT NR+ and LTE address different needs, DECT NR+ offers distinct advantages in specific areas, as summarized by the following table:
+
+.. table:: Feature comparison of DECT NR+ and LTE
+
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Feature                               | DECT NR+                          | LTE                        |
+   +=======================================+===================================+============================+
+   | License                               | License-exempt 1.9 GHz            | Requires licenses (varies) |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | IP based                              | No                                | Yes                        |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Coverage                              | Shorter, up to 3 km               | Wider, greater than 10 km  |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Performance in congested environments | Better                            | Poor                       |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Network flexibility                   | Star or mesh networks             | Primarily carrier-provided |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Power Consumption                     | Lower                             | Higher                     |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Data Rate (max)                       | 3.4 Mbps                          | Varies (typically higher)  |
```

Comment:
What does "Varies (typically higher" mean? If we are comparing to latest LTE high end modems, they are in the range of Gbits per sec

Reply:
I have removed this row.